everviz/main.py

```diff
@@ -0,0 +1,41 @@
+import functools
+import yaml
+import subprocess
+import os
+import shutil
+
+# Mock the hookimpl decorator
+# Allow everviz to be installed and have tests run without installing everest
+def mock_impl(func):
+    return func
+
+
+try:
+    from everest.plugins import hookimpl
+except ImportError:
+    hookimpl = mock_impl
+
+
+@hookimpl
+def visualize_data(everest_api):
+    config = webviz_config()
+
+    fname = "everviz_webviz_config.yml"
+    write_webviz_config(config, fname)
+
+    # The entry point of webviz is to call it from command line, and so we do.
+    if shutil.which("webviz"):
+        subprocess.call("webviz build {}".format(fname))
```

Comment:
include --theme equinor? The theme in itself is open, but only quasi open.

Reply:
I agree with adding the Equinor theme for now. We can remove it later very easily if we have problems with the "openes" of the package.

Reply:
Added